io.openems.edge.meter.api/src/io/openems/edge/meter/api/SinglePhaseMeter.java

```diff
@@ -25,39 +25,12 @@ public Doc doc() {
 	}
 
 	/**
-	 * Gets the Phase this ESS is connected to.
+	 * Gets the Phase this Meter is connected to.
 	 *
 	 * @return the Phase
 	 */
 	public SinglePhase getPhase();
 
-	/**
-	 * Initializes Channel listeners. Copies the Active-Power Phase-Channel value to
-	 * Active-Power Channel.
-	 *
-	 * @param meter the {@link AsymmetricMeter}
-	 * @param phase the {@link SinglePhase}
-	 */
-	public static void initializeCopyPhaseChannel(AsymmetricMeter meter, SinglePhase phase) {
```

Comment:
Was this never used?

Reply:
Apparently not :-)

For a new implementation I would like to refactor this feature. For the old method the actual phase had to be known from the beginning. But for some devices this might only be found during execution (e.g. by reading from Modbus). A new feature should be able to handle this dynamically. But for now it is not needed immediately.